compiler/il/VectorOperations.enum

```diff
@@ -739,6 +739,73 @@ VECTOR_OPERATION_MACRO(\
    /* .description             =    get vector element, returns scalar */ \
 )
 //
+// masked one-type operations
+//
+VECTOR_OPERATION_MACRO(\
+   /* .operation               = */ vmadd, \
+   /* .name                    = */ "vmadd", \
+   /* .properties1             = */ 0, \
+   /* .properties2             = */ ILProp2::ValueNumberShare | ILProp2::SupportedForPRE, \
```

Comment:
Add property to specify that it is masked, then create a function like 'isVectorMaskOpCode()'

Reply:
done